mods/ctf/ctf_map/nodes.lua

```diff
@@ -188,4 +189,23 @@ minetest.register_node("ctf_map:chest", {
 			minetest.show_formspec(player:get_player_name(), "", player:get_inventory_formspec())
 		end
 	end,
-})
+	on_rightclick = function(pos, node, clicker, itemstack, pointed_thing)
+		minetest.swap_node(pos, {name="ctf_map:chest_opened"})
+		local meta = minetest.get_meta(pos)
+		meta:set_string("infotext", chestv)
```

Comment:
```suggestion
		minetest.get_meta(pos):set_string("infotext", chestv)
```

No need for a local here unless you're planning to reuse it.